### PR TITLE
Added makemigrations command before migrate in dev file, fixed typos, translations

### PR DESCRIPTION
**Pull request by f-z**
Added makemigrations command before migrate in dev file
Fixed typo in register form in back-end
Fixed typo in last-thread in front-end

### CODE REVIEW

File: dev [hunk 1]
```diff
@@ -129,6 +129,8 @@ init() {
 init_in_docker() {
     require_in_docker
     wait_for_db
+    # make migrations
+    python manage.py makemigrations
```
Comment (rafalp) [inline dev]: This is wrong. Creating migrations should always be conscious decision by the developer.

**Comment by rafalp** [comment]
Please create separate PR's for separate improvements. I know its faster to just put all improvements in single PR, but when changes touch different areas of project, it makes impossible to apply those partially, and revert some if they break things.

**Comment by f-z** [comment]
Ok I will just submit the other fixes then

**Comment by rafalp** [comment]
Thanks! Also let me know if you have are present on misago forums or our discord, and I'll grant you `contributor` rank here 👍

**Comment by f-z** [comment]
yes I am my username is filipzof

**Pull request closed without merging**